pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/PinotControllerAuthResource.java

```diff
@@ -74,6 +74,11 @@ public class PinotControllerAuthResource {
   public boolean verify(@ApiParam(value = "Table name without type") @QueryParam("tableName") String tableName,
       @ApiParam(value = "API access type") @QueryParam("accessType") AccessType accessType,
       @ApiParam(value = "Endpoint URL") @QueryParam("endpointUrl") String endpointUrl) {
+
+    if (accessType == null) {
```

Comment:
Should we just add `@DefaultValue("READ")` to the `accessType`

Reply:
good with me. updated.